glean-core/src/metrics/timing_distribution.rs

```diff
@@ -232,7 +233,7 @@ impl TimingDistributionMetric {
     ///
     /// Please note that this assumes that the provided samples are already in the
     /// "unit" declared by the instance of the implementing metric type (e.g. if the
-    /// implementing class is a [TimingDistributionMetricType] and the instance this
+    /// implementing class is a [TimingDistributionMetric] and the instance this
```

Comment:
This sentence doesn't really make sense here. This is documentation in the metric type implementation itself, so it just refers to itself here?
I guess this doc was copied from the trait, but it shouldn't.

Reply:
Good point. Obviously the sentence about the unit of the input values is important, but the whole thing about implementation is irrelevant.